boa/util/evm.py

```diff
@@ -0,0 +1,40 @@
+from typing import Optional
+
+from eth.abc import ComputationAPI
+
+from boa import Env
+from boa.environment import Address
+from boa.util.exceptions import BoaError, strip_internal_frames
+
+
+class _EvmContract:
```

Comment:
this refactor was probably long overdue. i think the name should be `BaseEVMContract` though (no leading underscore!) and it should go somewhere like `boa/contracts/evm_contract.py`.